### ISSUE TITLE
Setting MTU and clustername?

### ISSUE THREAD

**Issue by xhejtman** [rancher/rke2#764]
Hello,

is it possible to set network MTU and clustername in rke2 as in rke1? 

network:
  mtu: 1400

cluster_name: 'xxx'

**Comment by brandond** [comment]
No, there are not any equivalent settings for RKE2.

**Comment by PatrickHuetter** [comment]
Just put the following yml-file into /var/lib/rancher/rke2/server/manifests/
`apiVersion: helm.cattle.io/v1
kind: HelmChartConfig
metadata:
  name: rke2-canal
  namespace: kube-system
spec:
  valuesContent: |-
    flannel:
      iface: "eth1"
    calico:
      vethuMTU: 1350
`
Afterwards networking with calico (default) will use MTU size of 1350.

**Comment by olljanat** [comment]
Btw. How exactly I should do this if I want to modify MTU/etc value for Calico? Based on https://docs.rke2.io/install/network_options/ those settings are under `installation` section which I understand on way that those should be in-place before Calico installation.

However it looks to be that RKE2 installation flush `/var/lib/rancher/rke2/server/manifests/` folder content if that exists before installation. Is only way to use `--cni none` on initial installation and change it afterwards?

PS. It was couple of week ago when I tested this logic last time so if that have been fixed recently it might be OK already but IMO documentation should be still clarified.
